docs/database-engine/availability-groups/windows/upgrading-always-on-availability-group-replica-instances.md

```diff
@@ -95,6 +95,11 @@ If necessary, you can perform an extra manual failover to return the AG to its o
 >
 > If `REQUIRED_SYNCHRONIZED_SECONDARIES_TO_COMMIT` is set to either `1` or `2`, the Primary replica may be unavailable for read/writes when a corresponding number of sync secondary replicas are not available during the update process.
 
+> [!NOTE]  
+> In case of Upgrade in-place on a secondary replica, which upgrades the instance to a newer version, the databases inside the AG will remain on state Synchronized / In Recovery. After a planned manual failover on the AG, the recovery will finish and the databases will be upgraded.
```

Comment:
```suggestion
> When you perform an in-place upgrade of a secondary replica to a newer version of SQL Server, the database inside the availability group remains in the **Synchronized / In Recovery** state until the availability group is manually failed over, which finishes the recovery and upgrades the database.   
```

Reply:
@asavioliMSFT please commit the suggestion if you agree my changes haven't technically changed the meaning of your addition. thanks!

Reply:
Thanks @MashaMSFT. I agree with your suggestion to change the text